README.md

```diff
@@ -120,6 +126,22 @@ cli:
     #    cli-flatten-payload-max-poly-as-resource-prop-count: 8
     #    # max properties allowed from flatten of sub-class as param
     #    cli-flatten-payload-max-poly-as-param-prop-count: 8
+
+    expand-operation:
+        # if true, operation with 'expand-operation-names' will be expanded into multiple 
+        # operation for ecah operation name. 
+        # Notice: expaneded operation's key is in formate: <OriginalOperation>#<ExpandedName>. 

```

Comment:
>Notice [](start = 10, length = 6)

it's also good to notice that if the new operation already exists, it will be ignored with a warning

Reply:
add cli property: `cli-operation-splitted`